methods/account/getAccountTransactions.md

```diff
@@ -260,9 +260,85 @@ The above command returns JSON structured like this:
   }
 ]
 ```
+### Example 4 of 6: Get transactions by number
 
+{% sample lang="bash" %}
+
+```bash
+curl -v -X GET  https://api.catapult.inetwork.com/v1/users/{user-id}/account/transations?number=+19199230192 \
+  -u {{apiToken}}:{secret} \
+  -H "Content-type: application/json" \
+```
+{% sample lang="js" %}
+
+```js
+//Get transactions limiting result
+//Promise
+var params = {
+  number: '+17208352955'
+}
+client.Account.getTransactions(params)
+.then(function (response) {
+    console.log(response.transactions);
+    if(response.hasNextPage) {
+        return response.getNextPage();
+    }
+    else {
+        return {transactions: []};
+    }
+})
+.then(function(response) {
+    console.log(response.transactions);
+});
+```
+
+{% sample lang="csharp" %}
+
+```csharp
+var transactions = await client.Account.GetTransactions(
+  new AccountTransactionQuery {
+    number = +13035659555
+  }
+);
+```
+
+{% sample lang="ruby" %}
+
+```ruby
+transactions = Account.get_transactions(client, {:number => "+19192130923"})
+```
+
+{% common %}
+The above command returns JSON structured like this:
+
+```json
```

Comment:
Add `[` and ` ]` around the JSON object here.  This is a list/array of json objects, enclosing in the `[]` lets us get better colors and formatting.

| Old | New |
|--|--|
| ![screen shot 2017-06-21 at 1 34 27 pm](https://user-images.githubusercontent.com/6805534/27398050-8081ff92-5686-11e7-9f4e-0bf2ea45db82.png) | ![screen shot 2017-06-21 at 1 34 39 pm](https://user-images.githubusercontent.com/6805534/27398056-86fd313e-5686-11e7-861d-cb98fad4c20d.png) |